external-adapter/src/validator.ts

```diff
@@ -32,20 +35,75 @@ export class Validator {
         }
       }
     } catch (error) {
-      const message = 'Error validating input.'
-      if (error instanceof AdapterError) this.error = error
-      else
-        this.error = new AdapterError({
-          jobRunID: this.validated.id,
-          statusCode: 400,
-          message,
-          cause: error,
-        })
-      logger.error(message, { error: this.error })
-      this.errored = Requester.errored(this.validated.id, this.error)
+      this.parseError(error)
     }
   }
 
+  validateOverrides() {
+    if (!this.input.data?.overrides) {
+      this.validated.overrides = this.formatOverride(presetSymbols)
+      return
+    }
+    try {
+      this.validated.overrides = this.formatOverride({
```

Comment:
I think we should deep merge here instead of shallow merging. I'm worried about some case where someone overrides say
```
{
  "tradermade": {
    "FTSE": "UK200"
  }
}
```
but now this overwrites all of the other overrides.

Should be pretty easy to do with lodash [.merge](https://lodash.com/docs/4.17.15#merge)